desc/equilibrium/utils.py

```diff
@@ -181,3 +188,165 @@ def parse_axis(axis, NFP=1, sym=True, surface=None):
     else:
         raise TypeError("Got unknown axis type {}".format(axis))
     return axis
+
+
+def scale_profile(profile, inner_rho, eq):
```

Comment:
I still think this whole function is unnecessary and could be replaced with `Profile.from_values` for all profile types. We should enforce `from_values` as a required method of every profile class. The only ones that are missing are `SplineProfile` (which is trivial to implement) and `TwoPowerProfile`